doc/manual/manual.tex

```diff
@@ -7795,6 +7795,43 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+
+\subsubsection{The Bunge et al. mantle convection experiments}
+
+\textit{This section was contributed by Cedric Thieulot and Bob Myhill.}
+
+Early mantle modelling studies of the 70's, 80's and 90's were often concerned with simple setups (Cartesian geometries, incompressible fluid, free slip boundary conditions) and investigated the influence of the Rayleigh number, the heating mode or the temperature dependence of the viscosity on temperature, pressure and/or strain rate \cite{youn74,buss75,buss79,BBC89,BC93,vavy93,burb97}. In this cookbook, we use the `simple' material model to reproduce the setup in \cite{burb96}, which reported that even modest increases in mantle viscosity with depth could have a marked effect on the style of mantle convection.
+
+Although the original article showcases results obtained in a 3D hollow sphere, we here run the models in an annular domain of inner radius R$_i$=3480 \si{\kilo\meter} and outer radius  R$_o$ = 6370 \si{\kilo\meter}.
+The surface temperature is set to $T_{\textit{surf}}$ = 1060 \si{\kelvin} and 
+the bottom temperature to $T_{\textit{cmb}}$ = 3450 \si{\kelvin}
+The gravity vector is radial and its magnitude is $g$ = 10 \si{\metre\per\second\squared}.
```

Comment:
```suggestion
The gravity vector is radial and its magnitude is $g = 10 \si{\metre\per\second\squared}$.
```

Reply:
Why do you favour putting the \si inside the $$? It works fine outside, and I prefer the kerning when it's not in the math environment.

Reply:
I think they logically go together. I can compromise on where the units are, but I think the `=10` should be inside the formula environment :-)

Reply:
Agreed on the `=10` ... that was really dumb of me.
If we put the \si{} into the math environment, we would need to manually separate number from unit with `\,` (otherwise we get 10m, rather than 10 m). So for now I've gone with the compromise option :)

Reply:
@cedrict, I think I made all of the suggested changes to the main text and bib text in the overleaf doc, if you want to copy them over.

Reply:
There is nothing ever "really dumb". We're all humans here.

Compromise sounds good to me.